src/ipc/ipc4/helper.c

```diff
@@ -436,7 +436,7 @@ int ipc_comp_connect(struct ipc *ipc, ipc_pipe_comp_connect *_connect)
 	struct comp_dev *sink;
 	struct ipc4_base_module_cfg source_src_cfg;
 	struct ipc4_base_module_cfg sink_src_cfg;
-	uint32_t flags;
+	uint32_t flags = 0;
```

Comment:
this variable seems to be just unused here... Then it should be removed, not initialised

Reply:
Yes, this code looks at least strange - explicit usage of a local variable in macro is not a good practice. 
Although it should not be a part of this PR I'll fix it

Reply:
thanks for addressing!